shared/readLFP.m

```diff
@@ -250,6 +248,9 @@
                     cfgtemp.trl(:,4)                = trialnr;
                     cfgtemp.trl(:,6)                = idir;
                     cfgtemp.trl(:,7)                = Stage;
+                    cfgtemp.trl(:,8)                = Startsample;
```

Comment:
just FYI: this ends up in data.sampleinfo (except offset). 
I was thinking of to deal with these fields though, because they are becoming more, and it becomes harder to make sure which column is which without checking. Now, fieldtrip has started to support MATLAB tables in trialinfo, so perhaps we should add these those (after redefining trials), which will also allow the naming of the fields, rather than just some arbitrary column number.
In short, i'm fine with this, but it will change again soon towards a more sustainable solution.